src/backend/tty.rs

```diff
@@ -320,6 +322,8 @@ impl Tty {
                 Ok::<_, anyhow::Error>((primary_node, primary_render_node))
             })?;
 
+        let disabled_nodes = disabled_nodes_from_config(&config.borrow());
```

Comment:
I feel like this setting should be live-reloaded. Since you find out at runtime that the device that you wanted to passthrough is taken by niri, and try to disable it.

Reply:
Implemented live-reloading.